searchAndSort.js

```diff
@@ -0,0 +1,62 @@
+import { moviesDatabase } from "./movieStorage.js";
+
+// Function to search for a keyword
+function searchMovies(keyword) {
+  return moviesDatabase.filter(movie =>
+    movie.title.toLowerCase().includes(keyword.toLowerCase()) ||
+    movie.description.toLowerCase().includes(keyword.toLowerCase()) ||
+    movie.director.toLowerCase().includes(keyword.toLowerCase()) ||
+    movie.actors.some(actor => actor.toLowerCase().includes(keyword.toLowerCase()))
+  ); 
+}
+
+// Function to sort the movie array by one or a few selected properties
+function sortMovies(properties) {
+  return [...moviesDatabase].sort((a, b) => {
+    for (let prop of properties) {
+      if (a[prop] < b[prop]) return -1;
+      if (a[prop] > b[prop]) return 1;
+    }
+    return 0;
+  });
+}
+
+// Function to filter by age rating
+function filterMoviesByAgeRating(ageRating) {
+  return moviesDatabase.filter(movie => 
+    movie.age_rating === ageRating
+  );
+}
+
+// Function to show movies over 120min
+const moviesOver2Hours = moviesDatabase
+.filter(movie => movie.movie_duration > 120)
```

Comment:
```suggestion
  .filter(movie => movie.movie_duration > 120)
```
Add indentation when you chain functions on a different line